src/components/Login/InputEmailAddr.js

```diff
@@ -46,15 +45,14 @@ class InputEmailAddr extends Component {
 
   onRequestSecretCodeCallback(result, error) {
     if (error) {
-      console.log(error);
-      if (error.code != ErrorMeta.ERR_NONE) {
+      if (error.msg) {
         Alert.alert(
           'Forgot password',
           'Please check email address that you inputted.',
         );
       }
     } else if (result) {
-      Actions.inputSecretCode({
+      Actions.findPassStep2({
```

Comment:
Okay seems it happened whileI resolving merge conflict... I applied wrong version